PDF_UA/WCAG-21-Complete.xml

```diff
@@ -180,7 +180,7 @@
             </error>
             <references/>
         </rule>
-        <rule object="SAP">
+        <rule object="SAP" tags="minor,structure,paragraph">
```

Comment:
I'd like to distinguish between "machine" and "human" rules. This one is clearly "human"